++ ls caseresult/ -ltr
++ wc -l
+ filename=case16
+ mkdir caseresult/case16
+ RESULT_DIR=/home/hadoop/git/physical_design/caseresult/case16
+ TEN=10
+ SLEEP_TIME=300
+ YCSB_DIR=/home/hadoop/ycsb-0.1.4
+ number_of_operations=5000000
+ number_of_threads=20
+ number_of_records=5000000
+ output_file_suffix=.dat
+ experiment_list_file=/home/hadoop/git/physical_design/case16.list
+ clean=false
+ create=false
+ load=false
+ random_read=false
+ scan=false
+ all=false
+ for i in '$@'
+ '[' all == all ']'
+ all=true
+ let counter=1
+ cd /home/hadoop/ycsb-0.1.4
++ pwd
+ echo /home/hadoop/ycsb-0.1.4
/home/hadoop/ycsb-0.1.4
+ read line
+ true
+ echo 'droping usertable...'
droping usertable...
+ hbase shell /home/hadoop/git/physical_design/delete_table
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hadoop/hbase-0.98.3-hadoop1/lib/slf4j-log4j12-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hadoop/hadoop-1.2.1/lib/slf4j-log4j12-1.4.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 row(s) in 33.5690 seconds

0 row(s) in 0.8030 seconds

0 row(s) in 0.0930 seconds

+ echo 'sleeping for 300 ...'
sleeping for 300 ...
+ sleep 300
+ true
+ echo 'creating usertable...'
creating usertable...
+ create_statement='create '\''usertable'\'', { NAME=>'\''family'\'', DATA_BLOCK_ENCODING=>'\''PREFIX_TREE'\''}, SPLITS => ['\''user1'\'', '\''user2'\'', '\''user3'\'', '\''user4'\'', '\''user5'\'', '\''user6'\'', '\''user7'\'', '\''user8'\'', '\''user9'\'']'
+ echo create ''\''usertable'\'',' '{' 'NAME=>'\''family'\'',' 'DATA_BLOCK_ENCODING=>'\''PREFIX_TREE'\''},' SPLITS '=>' '['\''user1'\'',' ''\''user2'\'',' ''\''user3'\'',' ''\''user4'\'',' ''\''user5'\'',' ''\''user6'\'',' ''\''user7'\'',' ''\''user8'\'',' ''\''user9'\'']'
+ echo 'exit '
+ create_table /home/hadoop/git/physical_design/caseresult/case16/create_1.script
+ echo 'creating table...'
creating table...
+ echo /home/hadoop/git/physical_design/caseresult/case16/create_1.script
/home/hadoop/git/physical_design/caseresult/case16/create_1.script
+ hbase shell /home/hadoop/git/physical_design/caseresult/case16/create_1.script
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hadoop/hbase-0.98.3-hadoop1/lib/slf4j-log4j12-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hadoop/hadoop-1.2.1/lib/slf4j-log4j12-1.4.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 row(s) in 1.6320 seconds

+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo 'loading data...'
loading data...
+ echo 'starting time: '
starting time: 
++ date +%s
+ echo 1404521825
1404521825
++ echo /home/hadoop/git/physical_design/caseresult/case16/load_1.dat
+ ./ld 5000000 20 /home/hadoop/git/physical_design/caseresult/case16/load_1.dat
Loading workload...
Starting test.
2014-07-04 17:57:05:862 0 sec: 0 operations; 
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/04 17:57:06 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x46f91a48, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/04 17:57:06 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x46f91a48 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/04 17:57:06 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/04 17:57:06 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/04 17:57:06 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x470400ccc6000f, negotiated timeout = 90000
2014-07-04 17:57:15:874 10 sec: 271554 operations; 27122.85 current ops/sec; [INSERT AverageLatency(us)=580.39] 
2014-07-04 17:57:25:876 20 sec: 309681 operations; 3811.94 current ops/sec; [INSERT AverageLatency(us)=383.8] 
2014-07-04 17:57:35:877 30 sec: 345956 operations; 3627.14 current ops/sec; [INSERT AverageLatency(us)=2200.11] 
2014-07-04 17:57:45:877 40 sec: 704508 operations; 35855.2 current ops/sec; [INSERT AverageLatency(us)=512.9] 
2014-07-04 17:57:55:878 50 sec: 1032707 operations; 32816.62 current ops/sec; [INSERT AverageLatency(us)=1038.21] 
2014-07-04 17:58:05:879 60 sec: 1322955 operations; 29021.9 current ops/sec; [INSERT AverageLatency(us)=1130.39] 
2014-07-04 17:58:15:879 70 sec: 1581784 operations; 25882.9 current ops/sec; [INSERT AverageLatency(us)=811.73] 
2014-07-04 17:58:25:880 80 sec: 1778285 operations; 19648.14 current ops/sec; [INSERT AverageLatency(us)=807.88] 
2014-07-04 17:58:35:881 90 sec: 1782378 operations; 409.26 current ops/sec; [INSERT AverageLatency(us)=7888.77] 
2014-07-04 17:58:47:325 101 sec: 1970308 operations; 16421.71 current ops/sec; [INSERT AverageLatency(us)=2000.34] 
2014-07-04 17:58:57:326 111 sec: 2182049 operations; 21171.98 current ops/sec; [INSERT AverageLatency(us)=947.42] 
2014-07-04 17:59:07:327 121 sec: 2353166 operations; 17111.7 current ops/sec; [INSERT AverageLatency(us)=1181.88] 
2014-07-04 17:59:17:327 131 sec: 2481549 operations; 12837.02 current ops/sec; [INSERT AverageLatency(us)=1378.21] 
2014-07-04 17:59:27:328 141 sec: 2600835 operations; 11927.41 current ops/sec; [INSERT AverageLatency(us)=1353.01] 
2014-07-04 17:59:37:329 151 sec: 2600835 operations; 0 current ops/sec;  
2014-07-04 17:59:47:329 161 sec: 2600835 operations; 0 current ops/sec;  
2014-07-04 17:59:57:330 171 sec: 2600835 operations; 0 current ops/sec;  
14/07/04 18:00:04 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 1913 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3211, waitTime=2714, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 10032 ms, replay 1913 ops.
14/07/04 18:00:04 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 1772 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3211, waitTime=2714, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 10070 ms, replay 1772 ops.
14/07/04 18:00:04 INFO client.AsyncProcess: #5, table=usertable, attempt=10/35 failed 1926 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3211, waitTime=2714, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:09 PDT 2014, retrying after 10025 ms, replay 1926 ops.
14/07/04 18:00:04 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 1870 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3229, waitTime=2580, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:08 PDT 2014, retrying after 10015 ms, replay 1870 ops.
14/07/04 18:00:04 INFO client.AsyncProcess: #11, table=usertable, attempt=10/35 failed 1666 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3229, waitTime=2580, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 10095 ms, replay 1666 ops.
14/07/04 18:00:05 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 2042 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3255, waitTime=2175, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:08 PDT 2014, retrying after 10023 ms, replay 2042 ops.
2014-07-04 18:00:07:331 181 sec: 2600835 operations; 0 current ops/sec;  
14/07/04 18:00:09 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 1950 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3276, waitTime=2028, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10083 ms, replay 1950 ops.
14/07/04 18:00:09 INFO client.AsyncProcess: #8, table=usertable, attempt=10/35 failed 2030 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3276, waitTime=2028, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10072 ms, replay 2030 ops.
14/07/04 18:00:09 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 1676 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3281, waitTime=2001, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10098 ms, replay 1676 ops.
14/07/04 18:00:09 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 1880 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3283, waitTime=2310, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10027 ms, replay 1880 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 32 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10034 ms, replay 32 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #13, table=usertable, attempt=10/35 failed 1804 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10089 ms, replay 1804 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 1910 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10006 ms, replay 1910 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 1811 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10031 ms, replay 1811 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 554 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10068 ms, replay 554 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 1908 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10062 ms, replay 1908 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #13, table=usertable, attempt=10/35 failed 561 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:12 PDT 2014, retrying after 10021 ms, replay 561 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 1875 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10086 ms, replay 1875 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 1817 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10079 ms, replay 1817 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10080 ms, replay 1883 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 1834 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10083 ms, replay 1834 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 622 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:14 PDT 2014, retrying after 10079 ms, replay 622 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 659 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10098 ms, replay 659 ops.
14/07/04 18:00:13 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 2025 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3298, waitTime=2328, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10086 ms, replay 2025 ops.
14/07/04 18:00:15 INFO client.AsyncProcess: #8, table=usertable, attempt=10/35 failed 601 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3314, waitTime=2854, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 10034 ms, replay 601 ops.
14/07/04 18:00:15 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 214 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3314, waitTime=2854, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:17 PDT 2014, retrying after 10009 ms, replay 214 ops.
14/07/04 18:00:15 INFO client.AsyncProcess: #8, table=usertable, attempt=10/35 failed 212 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3314, waitTime=2854, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10004 ms, replay 212 ops.
14/07/04 18:00:15 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 4 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3314, waitTime=2854, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10037 ms, replay 4 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #8, table=usertable, attempt=10/35 failed 76 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 10021 ms, replay 76 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 207 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 10049 ms, replay 207 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 1292 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10035 ms, replay 1292 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 240 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:17 PDT 2014, retrying after 10070 ms, replay 240 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 10 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 10096 ms, replay 10 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #11, table=usertable, attempt=10/35 failed 552 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10032 ms, replay 552 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #19, table=usertable, attempt=10/35 failed 1878 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10044 ms, replay 1878 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 1740 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10096 ms, replay 1740 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #5, table=usertable, attempt=11/35 failed 1926 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:09 PDT 2014, retrying after 10008 ms, replay 1926 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 failed 1772 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 10058 ms, replay 1772 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 1876 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10037 ms, replay 1876 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 1857 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 10009 ms, replay 1857 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 1904 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10015 ms, replay 1904 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 1946 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10040 ms, replay 1946 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 1837 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 10001 ms, replay 1837 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 1913 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 10022 ms, replay 1913 ops.
2014-07-04 18:00:17:331 191 sec: 2600835 operations; 0 current ops/sec;  
14/07/04 18:00:17 INFO client.AsyncProcess: #11, table=usertable, attempt=11/35 failed 1666 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 10081 ms, replay 1666 ops.
14/07/04 18:00:17 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 525 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10050 ms, replay 525 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 1789 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10087 ms, replay 1789 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 444 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10084 ms, replay 444 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #19, table=usertable, attempt=10/35 failed 1898 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10011 ms, replay 1898 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 2042 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3364, waitTime=2495, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:08 PDT 2014, retrying after 10060 ms, replay 2042 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 1885 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10017 ms, replay 1885 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 1872 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10025 ms, replay 1872 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 3 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10019 ms, replay 3 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 589 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10010 ms, replay 589 ops.
14/07/04 18:00:18 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 9 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10057 ms, replay 9 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 826 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10096 ms, replay 826 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 36 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10041 ms, replay 36 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #13, table=usertable, attempt=10/35 failed 697 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 10001 ms, replay 697 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 610 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10057 ms, replay 610 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 2 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10073 ms, replay 2 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 1087 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10051 ms, replay 1087 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 failed 1870 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:08 PDT 2014, retrying after 10029 ms, replay 1870 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 1855 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10050 ms, replay 1855 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 809 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10041 ms, replay 809 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 716 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10078 ms, replay 716 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 1 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10020 ms, replay 1 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 510 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10081 ms, replay 510 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 1284 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10070 ms, replay 1284 ops.
14/07/04 18:00:19 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 1 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10035 ms, replay 1 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 1908 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10002 ms, replay 1908 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #11, table=usertable, attempt=10/35 failed 80 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10057 ms, replay 80 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 861 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10024 ms, replay 861 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 558 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10094 ms, replay 558 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #11, table=usertable, attempt=10/35 failed 217 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10024 ms, replay 217 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 1236 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10091 ms, replay 1236 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 214 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10056 ms, replay 214 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 807 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:25 PDT 2014, retrying after 10029 ms, replay 807 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 1313 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10091 ms, replay 1313 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 209 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10083 ms, replay 209 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 1873 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10096 ms, replay 1873 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 13 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10046 ms, replay 13 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 82 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10054 ms, replay 82 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 74 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3367, waitTime=2703, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10054 ms, replay 74 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #11, table=usertable, attempt=10/35 failed 29 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3383, waitTime=2111, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10090 ms, replay 29 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 31 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3383, waitTime=2111, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10069 ms, replay 31 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 1860 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3383, waitTime=2111, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10074 ms, replay 1860 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #14, table=usertable, attempt=10/35 failed 6 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3383, waitTime=2111, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10022 ms, replay 6 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #8, table=usertable, attempt=11/35 failed 2030 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3383, waitTime=2111, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10001 ms, replay 2030 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 1676 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3383, waitTime=2111, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10062 ms, replay 1676 ops.
14/07/04 18:00:21 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 failed 1950 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3383, waitTime=2111, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10002 ms, replay 1950 ops.
14/07/04 18:00:23 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 failed 1880 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10084 ms, replay 1880 ops.
14/07/04 18:00:23 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 218 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10084 ms, replay 218 ops.
14/07/04 18:00:23 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 1299 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10080 ms, replay 1299 ops.
14/07/04 18:00:23 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 329 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10051 ms, replay 329 ops.
14/07/04 18:00:23 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 1887 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:25 PDT 2014, retrying after 10002 ms, replay 1887 ops.
14/07/04 18:00:23 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 577 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10065 ms, replay 577 ops.
14/07/04 18:00:24 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 206 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3402, waitTime=2063, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10027 ms, replay 206 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #13, table=usertable, attempt=11/35 failed 561 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:12 PDT 2014, retrying after 10076 ms, replay 561 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 1910 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10083 ms, replay 1910 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 1834 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10094 ms, replay 1834 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 622 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:14 PDT 2014, retrying after 10036 ms, replay 622 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 1817 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10057 ms, replay 1817 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 2025 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 10095 ms, replay 2025 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 failed 1908 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10083 ms, replay 1908 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #13, table=usertable, attempt=11/35 failed 1804 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10076 ms, replay 1804 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 failed 1875 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10070 ms, replay 1875 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 failed 1811 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10017 ms, replay 1811 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 32 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10029 ms, replay 32 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 failed 1883 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10078 ms, replay 1883 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 554 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10084 ms, replay 554 ops.
14/07/04 18:00:26 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 659 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:52826 remote=slave1/9.1.143.59:60020] on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 10067 ms, replay 659 ops.
2014-07-04 18:00:27:332 201 sec: 2600835 operations; 0 current ops/sec;  
14/07/04 18:00:27 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 4 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3421, waitTime=2704, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10082 ms, replay 4 ops.
14/07/04 18:00:27 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 214 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3421, waitTime=2704, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:17 PDT 2014, retrying after 10027 ms, replay 214 ops.
14/07/04 18:00:27 INFO client.AsyncProcess: #8, table=usertable, attempt=11/35 failed 212 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3421, waitTime=2704, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10011 ms, replay 212 ops.
14/07/04 18:00:27 INFO client.AsyncProcess: #8, table=usertable, attempt=11/35 failed 601 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3421, waitTime=2704, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 10076 ms, replay 601 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #11, table=usertable, attempt=11/35 failed 552 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10091 ms, replay 552 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 1946 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10042 ms, replay 1946 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 failed 1913 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 20013 ms, replay 1913 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #19, table=usertable, attempt=11/35 failed 1898 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10093 ms, replay 1898 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 10 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 10063 ms, replay 10 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 failed 1789 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10092 ms, replay 1789 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 1904 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10036 ms, replay 1904 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 444 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10046 ms, replay 444 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #11, table=usertable, attempt=12/35 failed 1666 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 20065 ms, replay 1666 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 failed 1292 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10005 ms, replay 1292 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #17, table=usertable, attempt=12/35 failed 1772 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014, retrying after 20162 ms, replay 1772 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 240 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:17 PDT 2014, retrying after 10059 ms, replay 240 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #19, table=usertable, attempt=11/35 failed 1878 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10006 ms, replay 1878 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 207 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 10000 ms, replay 207 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 1837 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 10076 ms, replay 1837 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 1740 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10072 ms, replay 1740 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 failed 2042 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:08 PDT 2014, retrying after 20147 ms, replay 2042 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 failed 1876 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10013 ms, replay 1876 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #5, table=usertable, attempt=12/35 failed 1926 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:09 PDT 2014, retrying after 20142 ms, replay 1926 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 1857 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 10026 ms, replay 1857 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #8, table=usertable, attempt=11/35 failed 76 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 10001 ms, replay 76 ops.
14/07/04 18:00:30 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 failed 525 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3427, waitTime=3195, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10046 ms, replay 525 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 9 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3451, waitTime=2119, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10042 ms, replay 9 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 1872 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3451, waitTime=2119, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10051 ms, replay 1872 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 589 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3451, waitTime=2119, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10025 ms, replay 589 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 1885 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3451, waitTime=2119, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10085 ms, replay 1885 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 3 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3451, waitTime=2119, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10037 ms, replay 3 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #13, table=usertable, attempt=11/35 failed 697 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3451, waitTime=2119, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 10001 ms, replay 697 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 36 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3457, waitTime=2009, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10050 ms, replay 36 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 2 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3458, waitTime=2189, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10035 ms, replay 2 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 1087 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3458, waitTime=2189, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10098 ms, replay 1087 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #6, table=usertable, attempt=12/35 failed 1870 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3458, waitTime=2189, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:08 PDT 2014, retrying after 20184 ms, replay 1870 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 failed 826 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3458, waitTime=2189, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10092 ms, replay 826 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 610 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3458, waitTime=2189, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 10002 ms, replay 610 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 1 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3463, waitTime=2274, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10088 ms, replay 1 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 failed 1855 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3463, waitTime=2274, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10027 ms, replay 1855 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 510 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3463, waitTime=2274, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 10092 ms, replay 510 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 809 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3463, waitTime=2274, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10051 ms, replay 809 ops.
14/07/04 18:00:31 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 716 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3463, waitTime=2274, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10094 ms, replay 716 ops.
14/07/04 18:00:32 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 1 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3468, waitTime=2387, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10026 ms, replay 1 ops.
14/07/04 18:00:32 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 failed 1284 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3468, waitTime=2387, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10088 ms, replay 1284 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #11, table=usertable, attempt=11/35 failed 217 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3472, waitTime=2416, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10060 ms, replay 217 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 failed 861 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3472, waitTime=2416, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10062 ms, replay 861 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 1236 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3472, waitTime=2416, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10024 ms, replay 1236 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 1908 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3472, waitTime=2416, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10003 ms, replay 1908 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 failed 558 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3472, waitTime=2416, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10081 ms, replay 558 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 214 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3472, waitTime=2416, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10009 ms, replay 214 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #11, table=usertable, attempt=11/35 failed 80 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3472, waitTime=2416, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10014 ms, replay 80 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 failed 807 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3472, waitTime=2416, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:25 PDT 2014, retrying after 10015 ms, replay 807 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 13 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3484, waitTime=2123, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10091 ms, replay 13 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 82 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3484, waitTime=2123, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10023 ms, replay 82 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 failed 209 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3484, waitTime=2123, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 10037 ms, replay 209 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 failed 1873 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3484, waitTime=2123, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10009 ms, replay 1873 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 failed 1313 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3484, waitTime=2123, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10040 ms, replay 1313 ops.
14/07/04 18:00:33 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 74 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3484, waitTime=2123, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 10069 ms, replay 74 ops.
14/07/04 18:00:34 INFO client.AsyncProcess: #14, table=usertable, attempt=11/35 failed 6 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3494, waitTime=2607, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10093 ms, replay 6 ops.
14/07/04 18:00:34 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 31 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3494, waitTime=2607, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10055 ms, replay 31 ops.
14/07/04 18:00:34 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 failed 1676 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3494, waitTime=2607, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 20146 ms, replay 1676 ops.
14/07/04 18:00:34 INFO client.AsyncProcess: #8, table=usertable, attempt=12/35 failed 2030 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3494, waitTime=2607, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20077 ms, replay 2030 ops.
14/07/04 18:00:34 INFO client.AsyncProcess: #11, table=usertable, attempt=11/35 failed 29 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3494, waitTime=2607, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10012 ms, replay 29 ops.
14/07/04 18:00:34 INFO client.AsyncProcess: #10, table=usertable, attempt=12/35 failed 1950 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3494, waitTime=2607, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 20093 ms, replay 1950 ops.
14/07/04 18:00:34 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 1860 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3494, waitTime=2607, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 10000 ms, replay 1860 ops.
14/07/04 18:00:36 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 329 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3503, waitTime=2491, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10052 ms, replay 329 ops.
14/07/04 18:00:36 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 failed 1299 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3503, waitTime=2491, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10015 ms, replay 1299 ops.
14/07/04 18:00:36 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 218 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3503, waitTime=2491, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10032 ms, replay 218 ops.
14/07/04 18:00:36 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 1887 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3503, waitTime=2491, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:25 PDT 2014, retrying after 10025 ms, replay 1887 ops.
14/07/04 18:00:36 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 577 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3503, waitTime=2491, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10006 ms, replay 577 ops.
14/07/04 18:00:36 INFO client.AsyncProcess: #12, table=usertable, attempt=12/35 failed 1880 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3503, waitTime=2491, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 20062 ms, replay 1880 ops.
14/07/04 18:00:36 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 206 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3511, waitTime=2243, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014, retrying after 10086 ms, replay 206 ops.
2014-07-04 18:00:37:332 211 sec: 2600835 operations; 0 current ops/sec;  
14/07/04 18:00:38 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 failed 32 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20138 ms, replay 32 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 failed 1817 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 20048 ms, replay 1817 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #3, table=usertable, attempt=12/35 failed 1811 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20010 ms, replay 1811 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #20, table=usertable, attempt=12/35 failed 1908 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20069 ms, replay 1908 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #13, table=usertable, attempt=12/35 failed 561 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:12 PDT 2014, retrying after 20090 ms, replay 561 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 failed 622 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:14 PDT 2014, retrying after 20043 ms, replay 622 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 554 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20012 ms, replay 554 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #15, table=usertable, attempt=12/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20107 ms, replay 1883 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 failed 1910 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20041 ms, replay 1910 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #13, table=usertable, attempt=12/35 failed 1804 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20016 ms, replay 1804 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #1, table=usertable, attempt=12/35 failed 1875 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20022 ms, replay 1875 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 659 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3516, waitTime=2058, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20183 ms, replay 659 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 1834 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3528, waitTime=2003, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20078 ms, replay 1834 ops.
14/07/04 18:00:38 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 2025 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3529, waitTime=2046, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 20078 ms, replay 2025 ops.
14/07/04 18:00:40 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 failed 214 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3530, waitTime=2347, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:17 PDT 2014, retrying after 20196 ms, replay 214 ops.
14/07/04 18:00:40 INFO client.AsyncProcess: #8, table=usertable, attempt=12/35 failed 601 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3530, waitTime=2347, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 20170 ms, replay 601 ops.
14/07/04 18:00:40 INFO client.AsyncProcess: #8, table=usertable, attempt=12/35 failed 212 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3530, waitTime=2347, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20085 ms, replay 212 ops.
14/07/04 18:00:40 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 failed 4 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3530, waitTime=2347, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20055 ms, replay 4 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #8, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014
14/07/04 18:00:42 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014
14/07/04 18:00:42 INFO client.AsyncProcess: #11, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:00:42 INFO client.AsyncProcess: #6, table=usertable, attempt=12/35 failed 1876 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3538, waitTime=2007, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20100 ms, replay 1876 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #19, table=usertable, attempt=12/35 failed 1878 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3538, waitTime=2007, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20084 ms, replay 1878 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #17, table=usertable, attempt=12/35 failed 1292 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3538, waitTime=2007, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20121 ms, replay 1292 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014
14/07/04 18:00:42 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 failed 1857 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 20163 ms, replay 1857 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 1904 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20000 ms, replay 1904 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 444 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20125 ms, replay 444 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 1740 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20198 ms, replay 1740 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #10, table=usertable, attempt=12/35 failed 525 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20143 ms, replay 525 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 failed 1837 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 20155 ms, replay 1837 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 failed 1946 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20014 ms, replay 1946 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #1, table=usertable, attempt=12/35 failed 1789 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20061 ms, replay 1789 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 240 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:17 PDT 2014, retrying after 20187 ms, replay 240 ops.
14/07/04 18:00:42 INFO client.AsyncProcess: #19, table=usertable, attempt=12/35 failed 1898 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3541, waitTime=2197, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20185 ms, replay 1898 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 9 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3553, waitTime=2056, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20095 ms, replay 9 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #13, table=usertable, attempt=12/35 failed 697 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3553, waitTime=2056, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 20149 ms, replay 697 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 failed 589 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3553, waitTime=2056, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20178 ms, replay 589 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014
14/07/04 18:00:43 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 3 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3553, waitTime=2056, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20117 ms, replay 3 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 failed 1872 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3553, waitTime=2056, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20078 ms, replay 1872 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 failed 36 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3558, waitTime=2317, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 20036 ms, replay 36 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 failed 1885 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3558, waitTime=2317, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20121 ms, replay 1885 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 610 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3558, waitTime=2317, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20096 ms, replay 610 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 1087 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3558, waitTime=2317, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 20015 ms, replay 1087 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #20, table=usertable, attempt=12/35 failed 826 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3558, waitTime=2317, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20062 ms, replay 826 ops.
14/07/04 18:00:43 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014
14/07/04 18:00:43 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #11, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 failed 510 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3564, waitTime=2512, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20064 ms, replay 510 ops.
14/07/04 18:00:44 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 716 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3564, waitTime=2512, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20024 ms, replay 716 ops.
14/07/04 18:00:44 INFO client.AsyncProcess: #1, table=usertable, attempt=12/35 failed 1855 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3564, waitTime=2512, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 20092 ms, replay 1855 ops.
14/07/04 18:00:44 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 failed 809 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3564, waitTime=2512, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20043 ms, replay 809 ops.
14/07/04 18:00:44 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #20, table=usertable, attempt=12/35 failed 1284 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3570, waitTime=2027, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20025 ms, replay 1284 ops.
14/07/04 18:00:44 INFO client.AsyncProcess: #15, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #15, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #11, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #11, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #1, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:25 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #14, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #10, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #10, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #20, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014
14/07/04 18:00:44 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:00:46 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014
14/07/04 18:00:46 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014
14/07/04 18:00:46 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014
14/07/04 18:00:46 INFO client.AsyncProcess: #1, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014
14/07/04 18:00:46 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:25 PDT 2014
14/07/04 18:00:46 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:24 PDT 2014
2014-07-04 18:00:47:333 221 sec: 2600835 operations; 0 current ops/sec;  
14/07/04 18:00:50 INFO client.AsyncProcess: #4, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014
14/07/04 18:00:51 INFO client.AsyncProcess: #11, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014
14/07/04 18:00:51 INFO client.AsyncProcess: #17, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:07 PDT 2014
14/07/04 18:00:51 INFO client.AsyncProcess: #5, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:09 PDT 2014
14/07/04 18:00:51 INFO client.AsyncProcess: #14, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:08 PDT 2014
14/07/04 18:00:52 INFO client.AsyncProcess: #6, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:08 PDT 2014
14/07/04 18:00:55 INFO client.AsyncProcess: #8, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:00:55 INFO client.AsyncProcess: #10, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014
14/07/04 18:00:55 INFO client.AsyncProcess: #7, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014
14/07/04 18:00:57 INFO client.AsyncProcess: #12, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014
2014-07-04 18:00:57:398 231 sec: 2746740 operations; 14496.27 current ops/sec; [INSERT AverageLatency(us)=5357.63] 
14/07/04 18:00:59 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:00:59 INFO client.AsyncProcess: #9, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:14 PDT 2014
14/07/04 18:00:59 INFO client.AsyncProcess: #4, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:00:59 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:00:59 INFO client.AsyncProcess: #13, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:12 PDT 2014
14/07/04 18:00:59 INFO client.AsyncProcess: #3, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:01:00 INFO client.AsyncProcess: #9, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:01:00 INFO client.AsyncProcess: #20, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:01:00 INFO client.AsyncProcess: #4, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:00 INFO client.AsyncProcess: #13, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:01:00 INFO client.AsyncProcess: #16, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014
14/07/04 18:01:01 INFO client.AsyncProcess: #8, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:01 INFO client.AsyncProcess: #1, table=usertable, attempt=13/35 failed 1875 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3864, waitTime=2165, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20110 ms, replay 1875 ops.
14/07/04 18:01:01 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 2025 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3864, waitTime=2165, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014, retrying after 20053 ms, replay 2025 ops.
14/07/04 18:01:01 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 failed 1834 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3864, waitTime=2165, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20162 ms, replay 1834 ops.
14/07/04 18:01:01 INFO client.AsyncProcess: #15, table=usertable, attempt=13/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3864, waitTime=2165, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014, retrying after 20193 ms, replay 1883 ops.
14/07/04 18:01:01 INFO client.AsyncProcess: #9, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:17 PDT 2014
14/07/04 18:01:01 INFO client.AsyncProcess: #8, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014
14/07/04 18:01:04 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:17 PDT 2014
14/07/04 18:01:05 INFO client.AsyncProcess: #19, table=usertable, attempt=13/35 failed 1878 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20050 ms, replay 1878 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #9, table=usertable, attempt=13/35 failed 1946 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20161 ms, replay 1946 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 444 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20041 ms, replay 444 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #7, table=usertable, attempt=13/35 failed 1857 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 20192 ms, replay 1857 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #6, table=usertable, attempt=13/35 failed 1876 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20008 ms, replay 1876 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 failed 1904 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20032 ms, replay 1904 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #10, table=usertable, attempt=13/35 failed 525 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20074 ms, replay 525 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 1740 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20156 ms, replay 1740 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #19, table=usertable, attempt=13/35 failed 1898 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20086 ms, replay 1898 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #16, table=usertable, attempt=13/35 failed 1837 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 20198 ms, replay 1837 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #1, table=usertable, attempt=13/35 failed 1789 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20116 ms, replay 1789 ops.
14/07/04 18:01:05 INFO client.AsyncProcess: #17, table=usertable, attempt=13/35 failed 1292 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3931, waitTime=2646, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20019 ms, replay 1292 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 failed 9 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20021 ms, replay 9 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 failed 3 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20093 ms, replay 3 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:06 INFO client.AsyncProcess: #14, table=usertable, attempt=13/35 failed 589 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20116 ms, replay 589 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #20, table=usertable, attempt=13/35 failed 826 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20050 ms, replay 826 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #16, table=usertable, attempt=13/35 failed 1872 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20146 ms, replay 1872 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #14, table=usertable, attempt=13/35 failed 36 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 20107 ms, replay 36 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 1087 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014, retrying after 20084 ms, replay 1087 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #7, table=usertable, attempt=13/35 failed 1885 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20040 ms, replay 1885 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #13, table=usertable, attempt=13/35 failed 697 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3954, waitTime=2782, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014, retrying after 20103 ms, replay 697 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 716 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3986, waitTime=2018, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20039 ms, replay 716 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #9, table=usertable, attempt=13/35 failed 809 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3989, waitTime=2012, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20084 ms, replay 809 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #9, table=usertable, attempt=13/35 failed 510 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3990, waitTime=2294, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20099 ms, replay 510 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #20, table=usertable, attempt=13/35 failed 1284 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3990, waitTime=2294, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20011 ms, replay 1284 ops.
14/07/04 18:01:06 INFO client.AsyncProcess: #1, table=usertable, attempt=13/35 failed 1855 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3990, waitTime=2294, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 20004 ms, replay 1855 ops.
2014-07-04 18:01:07:399 241 sec: 2864372 operations; 11762.02 current ops/sec; [INSERT AverageLatency(us)=3829.41] 
2014-07-04 18:01:17:400 251 sec: 3010896 operations; 14650.93 current ops/sec; [UPDATE AverageLatency(us)=2167158] [INSERT AverageLatency(us)=699.11] [CLEANUP AverageLatency(us)=2167496] 
14/07/04 18:01:22 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:10 PDT 2014
14/07/04 18:01:22 INFO client.AsyncProcess: #1, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:01:23 INFO client.AsyncProcess: #18, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:01:23 INFO client.AsyncProcess: #15, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:13 PDT 2014
14/07/04 18:01:26 INFO client.AsyncProcess: #14, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014
14/07/04 18:01:26 INFO client.AsyncProcess: #17, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:26 INFO client.AsyncProcess: #13, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:16 PDT 2014
14/07/04 18:01:26 INFO client.AsyncProcess: #14, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:26 INFO client.AsyncProcess: #18, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014
14/07/04 18:01:27 INFO client.AsyncProcess: #18, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:01:27 INFO client.AsyncProcess: #6, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
2014-07-04 18:01:27:453 261 sec: 3244424 operations; 23229.68 current ops/sec;  [INSERT AverageLatency(us)=2120.57]  
14/07/04 18:01:27 INFO client.AsyncProcess: #18, table=usertable, attempt=14/35 failed 1904 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4422, waitTime=2147, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20110 ms, replay 1904 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #10, table=usertable, attempt=14/35 failed 525 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4422, waitTime=2147, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20039 ms, replay 525 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #19, table=usertable, attempt=14/35 failed 1898 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4422, waitTime=2147, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20181 ms, replay 1898 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #1, table=usertable, attempt=14/35 failed 1789 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4422, waitTime=2147, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20049 ms, replay 1789 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 failed 1740 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4422, waitTime=2147, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20079 ms, replay 1740 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #9, table=usertable, attempt=14/35 failed 1946 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4422, waitTime=2147, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20118 ms, replay 1946 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 failed 444 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4422, waitTime=2147, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20085 ms, replay 444 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #19, table=usertable, attempt=14/35 failed 1878 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4422, waitTime=2147, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014, retrying after 20069 ms, replay 1878 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #7, table=usertable, attempt=14/35 failed 1857 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4432, waitTime=2196, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 20059 ms, replay 1857 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #16, table=usertable, attempt=14/35 failed 1837 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4432, waitTime=2196, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014, retrying after 20134 ms, replay 1837 ops.
14/07/04 18:01:27 INFO client.AsyncProcess: #20, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014
14/07/04 18:01:27 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:21 PDT 2014
14/07/04 18:01:27 INFO client.AsyncProcess: #7, table=usertable, attempt=14/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:01:28 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 failed 716 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4458, waitTime=2057, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20001 ms, replay 716 ops.
14/07/04 18:01:28 INFO client.AsyncProcess: #16, table=usertable, attempt=14/35 failed 1872 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4458, waitTime=2057, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20132 ms, replay 1872 ops.
14/07/04 18:01:28 INFO client.AsyncProcess: #9, table=usertable, attempt=14/35 failed 809 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4460, waitTime=2189, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20050 ms, replay 809 ops.
14/07/04 18:01:28 INFO client.AsyncProcess: #20, table=usertable, attempt=14/35 failed 1284 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4469, waitTime=2090, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014, retrying after 20163 ms, replay 1284 ops.
14/07/04 18:01:28 INFO client.AsyncProcess: #1, table=usertable, attempt=14/35 failed 1855 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4469, waitTime=2090, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014, retrying after 20133 ms, replay 1855 ops.
14/07/04 18:01:28 INFO client.AsyncProcess: #9, table=usertable, attempt=14/35 failed 510 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4469, waitTime=2090, rpcTimetout=2000 on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014, retrying after 20001 ms, replay 510 ops.
2014-07-04 18:01:37:712 271 sec: 3476407 operations; 22612.63 current ops/sec; [UPDATE AverageLatency(us)=5582970] [INSERT AverageLatency(us)=394.7] [CLEANUP AverageLatency(us)=5584624] 
14/07/04 18:01:47 INFO client.AsyncProcess: #10, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
2014-07-04 18:01:47:824 281 sec: 3669095 operations; 19055.38 current ops/sec; [UPDATE AverageLatency(us)=4365763] [INSERT AverageLatency(us)=646.69] [CLEANUP AverageLatency(us)=4365840] 
14/07/04 18:01:47 INFO client.AsyncProcess: #2, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014
14/07/04 18:01:47 INFO client.AsyncProcess: #1, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:01:47 INFO client.AsyncProcess: #2, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:48 INFO client.AsyncProcess: #19, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:48 INFO client.AsyncProcess: #18, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:48 INFO client.AsyncProcess: #9, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:18 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #2, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #7, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #16, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:19 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #19, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #9, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #9, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #20, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:22 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #16, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:23 PDT 2014
14/07/04 18:01:49 INFO client.AsyncProcess: #1, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404521467638, tracking started Fri Jul 04 17:59:20 PDT 2014
2014-07-04 18:01:58:038 292 sec: 3866452 operations; 19322.2 current ops/sec; [UPDATE AverageLatency(us)=6016051] [INSERT AverageLatency(us)=4658.33] [CLEANUP AverageLatency(us)=6016077.67] 
2014-07-04 18:02:08:039 302 sec: 4149844 operations; 28336.37 current ops/sec;  [INSERT AverageLatency(us)=483.75]  
2014-07-04 18:02:18:039 312 sec: 4321073 operations; 17122.9 current ops/sec; [UPDATE AverageLatency(us)=10462829] [INSERT AverageLatency(us)=782.19] [CLEANUP AverageLatency(us)=10462846] 
2014-07-04 18:02:28:042 322 sec: 4490017 operations; 16889.33 current ops/sec; [UPDATE AverageLatency(us)=6178049] [INSERT AverageLatency(us)=679.14] [CLEANUP AverageLatency(us)=6178075.5] 
2014-07-04 18:02:38:043 332 sec: 4730617 operations; 24057.59 current ops/sec; [UPDATE AverageLatency(us)=6453324] [INSERT AverageLatency(us)=364.93] [CLEANUP AverageLatency(us)=6453350] 
2014-07-04 18:02:48:044 342 sec: 4913363 operations; 18272.77 current ops/sec; [UPDATE AverageLatency(us)=2868807.71] [INSERT AverageLatency(us)=341.06] [CLEANUP AverageLatency(us)=2868831.14] 
2014-07-04 18:02:52:511 346 sec: 5000000 operations; 19394.9 current ops/sec; [UPDATE AverageLatency(us)=1025152.67] [INSERT AverageLatency(us)=99.11] [CLEANUP AverageLatency(us)=1025176.67] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404522173
1404522173
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo 'random read...'
random read...
+ echo 'starting time: '
starting time: 
++ date +%s
+ echo 1404522183
1404522183
++ echo /home/hadoop/git/physical_design/caseresult/case16/random_read_1.dat
+ ./random_read 5000000 20 /home/hadoop/git/physical_design/caseresult/case16/random_read_1.dat
Loading workload...
Starting test.
2014-07-04 18:03:05:968 0 sec: 0 operations; 
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/04 18:03:06 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x4ef888ea, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/04 18:03:06 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x4ef888ea connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/04 18:03:06 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/04 18:03:06 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/04 18:03:07 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x1470400c83d0009, negotiated timeout = 90000
2014-07-04 18:03:15:979 10 sec: 60007 operations; 5964.91 current ops/sec; [READ AverageLatency(us)=3301.44] 
2014-07-04 18:03:25:980 20 sec: 205328 operations; 14530.65 current ops/sec; [READ AverageLatency(us)=1371.31] 
2014-07-04 18:03:35:981 30 sec: 348085 operations; 14274.27 current ops/sec; [READ AverageLatency(us)=1394.69] 
2014-07-04 18:03:45:984 40 sec: 486488 operations; 13836.15 current ops/sec; [READ AverageLatency(us)=1440.61] 
2014-07-04 18:03:55:985 50 sec: 629821 operations; 14333.3 current ops/sec; [READ AverageLatency(us)=1391.43] 
2014-07-04 18:04:05:985 60 sec: 804524 operations; 17468.55 current ops/sec; [READ AverageLatency(us)=1139.94] 
2014-07-04 18:04:15:986 70 sec: 978162 operations; 17362.06 current ops/sec; [READ AverageLatency(us)=1146.94] 
2014-07-04 18:04:25:986 80 sec: 1151947 operations; 17378.5 current ops/sec; [READ AverageLatency(us)=1143.71] 
2014-07-04 18:04:35:987 90 sec: 1325363 operations; 17339.87 current ops/sec; [READ AverageLatency(us)=1150.61] 
2014-07-04 18:04:45:992 100 sec: 1498086 operations; 17263.67 current ops/sec; [READ AverageLatency(us)=1153.41] 
2014-07-04 18:04:55:992 110 sec: 1668850 operations; 17076.4 current ops/sec; [READ AverageLatency(us)=1166.21] 
2014-07-04 18:05:05:993 120 sec: 1841877 operations; 17300.97 current ops/sec; [READ AverageLatency(us)=1151.02] 
2014-07-04 18:05:15:997 130 sec: 2015398 operations; 17345.16 current ops/sec; [READ AverageLatency(us)=1147.76] 
2014-07-04 18:05:25:998 140 sec: 2186422 operations; 17100.69 current ops/sec; [READ AverageLatency(us)=1164.76] 
2014-07-04 18:05:35:998 150 sec: 2356024 operations; 16960.2 current ops/sec; [READ AverageLatency(us)=1170.12] 
2014-07-04 18:05:45:999 160 sec: 2527485 operations; 17144.39 current ops/sec; [READ AverageLatency(us)=1165.65] 
2014-07-04 18:05:56:000 170 sec: 2697837 operations; 17033.5 current ops/sec; [READ AverageLatency(us)=1169.08] 
2014-07-04 18:06:06:003 180 sec: 2868236 operations; 17034.79 current ops/sec; [READ AverageLatency(us)=1166.46] 
2014-07-04 18:06:16:004 190 sec: 3038387 operations; 17013.4 current ops/sec; [READ AverageLatency(us)=1173.15] 
2014-07-04 18:06:26:005 200 sec: 3207306 operations; 16890.21 current ops/sec; [READ AverageLatency(us)=1179.06] 
2014-07-04 18:06:36:005 210 sec: 3375935 operations; 16862.9 current ops/sec; [READ AverageLatency(us)=1181.11] 
2014-07-04 18:06:46:006 220 sec: 3540480 operations; 16452.85 current ops/sec; [READ AverageLatency(us)=1210.52] 
2014-07-04 18:06:56:007 230 sec: 3704641 operations; 16414.46 current ops/sec; [READ AverageLatency(us)=1213.34] 
2014-07-04 18:07:06:007 240 sec: 3875557 operations; 17091.6 current ops/sec; [READ AverageLatency(us)=1161.25] 
2014-07-04 18:07:16:008 250 sec: 4046672 operations; 17109.79 current ops/sec; [READ AverageLatency(us)=1167.88] 
2014-07-04 18:07:26:009 260 sec: 4216974 operations; 17028.5 current ops/sec; [READ AverageLatency(us)=1169.39] 
2014-07-04 18:07:36:009 270 sec: 4388077 operations; 17110.3 current ops/sec; [READ AverageLatency(us)=1164.01] 
2014-07-04 18:07:46:010 280 sec: 4558224 operations; 17013 current ops/sec; [READ AverageLatency(us)=1170.55] 
2014-07-04 18:07:56:010 290 sec: 4729953 operations; 17172.9 current ops/sec; [READ AverageLatency(us)=1159.74] 
2014-07-04 18:08:06:011 300 sec: 4902561 operations; 17259.07 current ops/sec; [READ AverageLatency(us)=1153.74] 
2014-07-04 18:08:11:922 306 sec: 5000000 operations; 16484.35 current ops/sec; [UPDATE AverageLatency(us)=11.85] [READ AverageLatency(us)=1140.32] [CLEANUP AverageLatency(us)=161.25] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404522492
1404522492
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloada...
workloada...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404522502
1404522502
++ echo /home/hadoop/git/physical_design/caseresult/case16/workloada_1.dat
+ ./workloada 5000000 20 /home/hadoop/git/physical_design/caseresult/case16/workloada_1.dat
Loading workload...
Starting test.
2014-07-04 18:08:22:980 0 sec: 0 operations; 
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/04 18:08:23 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x3cfbef54, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/04 18:08:23 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x3cfbef54 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/04 18:08:23 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/04 18:08:23 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/04 18:08:23 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x470400ccc60010, negotiated timeout = 90000
2014-07-04 18:08:32:983 10 sec: 29141 operations; 2913.23 current ops/sec; [INSERT AverageLatency(us)=459.63] [READ AverageLatency(us)=3923.77] 
2014-07-04 18:08:42:984 20 sec: 146181 operations; 11702.83 current ops/sec; [INSERT AverageLatency(us)=17.01] [READ AverageLatency(us)=1708.44] 
2014-07-04 18:08:52:985 30 sec: 280749 operations; 13455.45 current ops/sec; [INSERT AverageLatency(us)=12.72] [READ AverageLatency(us)=1624.55] 
2014-07-04 18:09:02:986 40 sec: 425718 operations; 14495.45 current ops/sec; [INSERT AverageLatency(us)=12.78] [READ AverageLatency(us)=1513.46] 
2014-07-04 18:09:12:987 50 sec: 571198 operations; 14546.55 current ops/sec; [INSERT AverageLatency(us)=13.07] [READ AverageLatency(us)=1508.45] 
2014-07-04 18:09:22:987 60 sec: 711729 operations; 14053.1 current ops/sec; [INSERT AverageLatency(us)=11.94] [READ AverageLatency(us)=1566.24] 
2014-07-04 18:09:32:988 70 sec: 865080 operations; 15333.57 current ops/sec; [INSERT AverageLatency(us)=13.68] [READ AverageLatency(us)=1436.31] 
2014-07-04 18:09:42:989 80 sec: 965050 operations; 9996 current ops/sec; [INSERT AverageLatency(us)=50.92] [READ AverageLatency(us)=2205.25] 
2014-07-04 18:09:52:990 90 sec: 1093560 operations; 12849.72 current ops/sec; [INSERT AverageLatency(us)=17.92] [READ AverageLatency(us)=1720.57] 
2014-07-04 18:10:02:991 100 sec: 1233067 operations; 13949.31 current ops/sec; [INSERT AverageLatency(us)=17.04] [READ AverageLatency(us)=1576.71] 
2014-07-04 18:10:12:991 110 sec: 1374901 operations; 14183.4 current ops/sec; [INSERT AverageLatency(us)=21.05] [READ AverageLatency(us)=1560.14] 
2014-07-04 18:10:22:992 120 sec: 1522989 operations; 14807.32 current ops/sec; [INSERT AverageLatency(us)=15.15] [READ AverageLatency(us)=1441.31] 
2014-07-04 18:10:32:993 130 sec: 1660683 operations; 13768.02 current ops/sec; [INSERT AverageLatency(us)=15.7] [READ AverageLatency(us)=1652.68] 
2014-07-04 18:10:42:994 140 sec: 1811821 operations; 15112.29 current ops/sec; [INSERT AverageLatency(us)=16.98] [READ AverageLatency(us)=1459.29] 
2014-07-04 18:10:52:995 150 sec: 1904273 operations; 9244.28 current ops/sec; [INSERT AverageLatency(us)=47.8] [READ AverageLatency(us)=2389.34] 
2014-07-04 18:11:02:996 160 sec: 2015701 operations; 11141.69 current ops/sec; [INSERT AverageLatency(us)=21.03] [READ AverageLatency(us)=1974.36] 
2014-07-04 18:11:12:996 170 sec: 2160055 operations; 14435.4 current ops/sec; [INSERT AverageLatency(us)=16.68] [READ AverageLatency(us)=1537.13] 
2014-07-04 18:11:22:998 180 sec: 2316397 operations; 15632.64 current ops/sec; [INSERT AverageLatency(us)=14.72] [READ AverageLatency(us)=1412.83] 
2014-07-04 18:11:32:998 190 sec: 2488788 operations; 17237.38 current ops/sec; [INSERT AverageLatency(us)=20.18] [READ AverageLatency(us)=1276.68] 
2014-07-04 18:11:42:999 200 sec: 2657880 operations; 16907.51 current ops/sec; [INSERT AverageLatency(us)=15.51] [READ AverageLatency(us)=1304.95] 
2014-07-04 18:11:53:000 210 sec: 2802056 operations; 14416.16 current ops/sec; [INSERT AverageLatency(us)=19.17] [READ AverageLatency(us)=1532.44] 
2014-07-04 18:12:03:000 220 sec: 2911287 operations; 10923.1 current ops/sec; [INSERT AverageLatency(us)=26.96] [READ AverageLatency(us)=2020.72] 
2014-07-04 18:12:13:001 230 sec: 3076046 operations; 16474.25 current ops/sec; [INSERT AverageLatency(us)=14.55] [READ AverageLatency(us)=1338.82] 
2014-07-04 18:12:23:002 240 sec: 3239882 operations; 16381.96 current ops/sec; [INSERT AverageLatency(us)=17.03] [READ AverageLatency(us)=1347.21] 
2014-07-04 18:12:33:003 250 sec: 3409459 operations; 16956 current ops/sec; [INSERT AverageLatency(us)=16.97] [READ AverageLatency(us)=1300.16] 
2014-07-04 18:12:43:004 260 sec: 3582959 operations; 17348.27 current ops/sec; [INSERT AverageLatency(us)=15.44] [READ AverageLatency(us)=1270.69] 
2014-07-04 18:12:53:004 270 sec: 3738758 operations; 15579.9 current ops/sec; [INSERT AverageLatency(us)=18.6] [READ AverageLatency(us)=1400.07] 
2014-07-04 18:13:03:005 280 sec: 3852236 operations; 11346.67 current ops/sec; [INSERT AverageLatency(us)=22.22] [READ AverageLatency(us)=1972.62] 
2014-07-04 18:13:13:006 290 sec: 4019214 operations; 16696.13 current ops/sec; [INSERT AverageLatency(us)=16.65] [READ AverageLatency(us)=1321.26] 
2014-07-04 18:13:23:007 300 sec: 4187172 operations; 16794.12 current ops/sec; [INSERT AverageLatency(us)=17.22] [READ AverageLatency(us)=1311.87] 
2014-07-04 18:13:33:007 310 sec: 4354806 operations; 16763.4 current ops/sec; [INSERT AverageLatency(us)=16.14] [READ AverageLatency(us)=1316.01] 
2014-07-04 18:13:43:008 320 sec: 4528830 operations; 17400.66 current ops/sec; [INSERT AverageLatency(us)=15.14] [READ AverageLatency(us)=1267.38] 
2014-07-04 18:13:53:009 330 sec: 4678404 operations; 14955.9 current ops/sec; [INSERT AverageLatency(us)=18.91] [READ AverageLatency(us)=1477.69] 
2014-07-04 18:14:03:010 340 sec: 4806131 operations; 12771.42 current ops/sec; [INSERT AverageLatency(us)=21.71] [READ AverageLatency(us)=1730.78] 
2014-07-04 18:14:13:011 350 sec: 4953013 operations; 14686.73 current ops/sec; [INSERT AverageLatency(us)=16.58] [READ AverageLatency(us)=1504.39] 
2014-07-04 18:14:17:889 354 sec: 5000000 operations; 9632.43 current ops/sec; [UPDATE AverageLatency(us)=141784] [INSERT AverageLatency(us)=15.06] [READ AverageLatency(us)=1533.08] [CLEANUP AverageLatency(us)=141986.45] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404522858
1404522858
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloadb...
workloadb...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404522868
1404522868
++ echo /home/hadoop/git/physical_design/caseresult/case16/workloadb_1.dat
+ ./workloadb 5000000 20 /home/hadoop/git/physical_design/caseresult/case16/workloadb_1.dat
Loading workload...
Starting test.
2014-07-04 18:14:29:117 0 sec: 0 operations; 
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/04 18:14:29 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0xfaaf84c, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/04 18:14:29 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0xfaaf84c connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/04 18:14:29 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/04 18:14:29 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/04 18:14:29 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x1470400c83d000a, negotiated timeout = 90000
2014-07-04 18:14:39:119 10 sec: 26107 operations; 2610.18 current ops/sec; [INSERT AverageLatency(us)=1069.63] [READ AverageLatency(us)=5241.83] 
2014-07-04 18:14:49:293 20 sec: 185559 operations; 15672.5 current ops/sec; [INSERT AverageLatency(us)=37.51] [READ AverageLatency(us)=2072.96] 
2014-07-04 18:14:59:399 30 sec: 344514 operations; 15728.77 current ops/sec; [INSERT AverageLatency(us)=50.39] [READ AverageLatency(us)=2473.4] 
2014-07-04 18:15:09:400 40 sec: 466261 operations; 12173.48 current ops/sec; [INSERT AverageLatency(us)=20.56] [READ AverageLatency(us)=3238.81] 
2014-07-04 18:15:19:408 50 sec: 582841 operations; 11648.68 current ops/sec; [INSERT AverageLatency(us)=17.74] [READ AverageLatency(us)=3369.48] 
2014-07-04 18:15:29:409 60 sec: 747506 operations; 16464.85 current ops/sec; [INSERT AverageLatency(us)=24.52] [READ AverageLatency(us)=2373.74] 
2014-07-04 18:15:39:410 70 sec: 921225 operations; 17370.16 current ops/sec; [INSERT AverageLatency(us)=18.71] [READ AverageLatency(us)=2262.96] 
2014-07-04 18:15:49:411 80 sec: 1093129 operations; 17188.68 current ops/sec; [INSERT AverageLatency(us)=18.13] [READ AverageLatency(us)=2295.76] 
2014-07-04 18:15:59:412 90 sec: 1243447 operations; 15030.3 current ops/sec; [INSERT AverageLatency(us)=16.95] [READ AverageLatency(us)=2621.92] 
2014-07-04 18:16:09:413 100 sec: 1422154 operations; 17868.91 current ops/sec; [INSERT AverageLatency(us)=17.91] [READ AverageLatency(us)=2211.33] 
2014-07-04 18:16:19:413 110 sec: 1602164 operations; 18001 current ops/sec; [INSERT AverageLatency(us)=20.92] [READ AverageLatency(us)=2184.5] 
2014-07-04 18:16:29:414 120 sec: 1782342 operations; 18016 current ops/sec; [INSERT AverageLatency(us)=18.55] [READ AverageLatency(us)=2185.41] 
2014-07-04 18:16:39:415 130 sec: 1940366 operations; 15800.82 current ops/sec; [INSERT AverageLatency(us)=21.39] [READ AverageLatency(us)=2486.31] 
2014-07-04 18:16:49:415 140 sec: 2081498 operations; 14113.2 current ops/sec; [INSERT AverageLatency(us)=20.05] [READ AverageLatency(us)=2797.18] 
2014-07-04 18:16:59:416 150 sec: 2239859 operations; 15834.52 current ops/sec; [INSERT AverageLatency(us)=30.68] [READ AverageLatency(us)=2472.12] 
2014-07-04 18:17:09:421 160 sec: 2285639 operations; 4575.71 current ops/sec; [INSERT AverageLatency(us)=65.72] [READ AverageLatency(us)=8607.3] 
2014-07-04 18:17:19:422 170 sec: 2430958 operations; 14530.45 current ops/sec; [INSERT AverageLatency(us)=23.35] [READ AverageLatency(us)=2732.45] 
2014-07-04 18:17:29:553 180 sec: 2537837 operations; 10549.7 current ops/sec; [INSERT AverageLatency(us)=26.23] [READ AverageLatency(us)=3722.2] 
2014-07-04 18:17:39:554 190 sec: 2672446 operations; 13459.55 current ops/sec; [INSERT AverageLatency(us)=17.5] [READ AverageLatency(us)=2985.27] 
2014-07-04 18:17:49:555 200 sec: 2827676 operations; 15521.45 current ops/sec; [INSERT AverageLatency(us)=15.99] [READ AverageLatency(us)=2545.15] 
2014-07-04 18:17:59:670 210 sec: 2993518 operations; 16395.65 current ops/sec; [INSERT AverageLatency(us)=19.4] [READ AverageLatency(us)=2363.35] 
2014-07-04 18:18:09:671 220 sec: 3122893 operations; 12936.21 current ops/sec; [INSERT AverageLatency(us)=17.01] [READ AverageLatency(us)=3127.53] 
2014-07-04 18:18:19:671 230 sec: 3223145 operations; 10025.2 current ops/sec; [INSERT AverageLatency(us)=19.75] [READ AverageLatency(us)=3928.14] 
2014-07-04 18:18:29:672 240 sec: 3364403 operations; 14124.39 current ops/sec; [INSERT AverageLatency(us)=24.23] [READ AverageLatency(us)=2697.36] 
2014-07-04 18:18:39:673 250 sec: 3470754 operations; 10634.04 current ops/sec; [INSERT AverageLatency(us)=21.15] [READ AverageLatency(us)=3888.87] 
2014-07-04 18:18:49:674 260 sec: 3614471 operations; 14371.7 current ops/sec; [INSERT AverageLatency(us)=21.33] [READ AverageLatency(us)=2737.65] 
2014-07-04 18:18:59:674 270 sec: 3753129 operations; 13864.41 current ops/sec; [INSERT AverageLatency(us)=18.74] [READ AverageLatency(us)=2766.72] 
2014-07-04 18:19:09:676 280 sec: 3872619 operations; 11946.61 current ops/sec; [INSERT AverageLatency(us)=18.43] [READ AverageLatency(us)=3438.98] 
2014-07-04 18:19:19:677 290 sec: 3975532 operations; 10290.27 current ops/sec; [INSERT AverageLatency(us)=22.92] [READ AverageLatency(us)=3859.33] 
2014-07-04 18:19:29:678 300 sec: 4125984 operations; 15043.7 current ops/sec; [INSERT AverageLatency(us)=15.96] [READ AverageLatency(us)=2622.85] 
2014-07-04 18:19:39:682 310 sec: 4282825 operations; 15677.83 current ops/sec; [INSERT AverageLatency(us)=20.26] [READ AverageLatency(us)=2499.63] 
2014-07-04 18:19:49:682 320 sec: 4383390 operations; 10056.5 current ops/sec; [INSERT AverageLatency(us)=23.55] [READ AverageLatency(us)=3974.4] 
2014-07-04 18:19:59:684 330 sec: 4523443 operations; 14003.9 current ops/sec; [INSERT AverageLatency(us)=25.16] [READ AverageLatency(us)=2834.94] 
2014-07-04 18:20:09:684 340 sec: 4688347 operations; 16488.75 current ops/sec; [INSERT AverageLatency(us)=18.83] [READ AverageLatency(us)=2388.01] 
2014-07-04 18:20:19:685 350 sec: 4844278 operations; 15591.54 current ops/sec; [INSERT AverageLatency(us)=19.76] [READ AverageLatency(us)=2527.29] 
2014-07-04 18:20:29:686 360 sec: 4949477 operations; 10518.85 current ops/sec; [INSERT AverageLatency(us)=21.06] [READ AverageLatency(us)=3769.88] 
2014-07-04 18:20:35:837 366 sec: 5000000 operations; 8213.79 current ops/sec; [UPDATE AverageLatency(us)=299432.85] [INSERT AverageLatency(us)=24.4] [READ AverageLatency(us)=2894.84] [CLEANUP AverageLatency(us)=299613.35] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404523236
1404523236
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloadc...
workloadc...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404523246
1404523246
++ echo /home/hadoop/git/physical_design/caseresult/case16/workloadc_1.dat
+ ./workloadc 5000000 20 /home/hadoop/git/physical_design/caseresult/case16/workloadc_1.dat
Loading workload...
Starting test.
2014-07-04 18:20:48:420 0 sec: 0 operations; 
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/04 18:20:49 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0xc2c915f, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/04 18:20:49 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/04 18:20:49 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0xc2c915f connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/04 18:20:49 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/04 18:20:49 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x470400ccc60011, negotiated timeout = 90000
2014-07-04 18:20:58:422 10 sec: 134833 operations; 13480.6 current ops/sec; [INSERT AverageLatency(us)=280.19] [READ AverageLatency(us)=9275.41] 
2014-07-04 18:21:08:424 20 sec: 318747 operations; 18387.72 current ops/sec; [INSERT AverageLatency(us)=51.35] [READ AverageLatency(us)=10233.6] 
2014-07-04 18:21:18:425 30 sec: 520441 operations; 20169.4 current ops/sec; [INSERT AverageLatency(us)=19.76] [READ AverageLatency(us)=9770.35] 
2014-07-04 18:21:28:425 40 sec: 727327 operations; 20686.53 current ops/sec; [INSERT AverageLatency(us)=25.59] [READ AverageLatency(us)=9193.04] 
2014-07-04 18:21:38:426 50 sec: 904769 operations; 17742.43 current ops/sec; [INSERT AverageLatency(us)=16.86] [READ AverageLatency(us)=10944.72] 
2014-07-04 18:21:48:427 60 sec: 1026836 operations; 12205.48 current ops/sec; [INSERT AverageLatency(us)=16.93] [READ AverageLatency(us)=15958.18] 
2014-07-04 18:21:58:428 70 sec: 1150270 operations; 12343.4 current ops/sec; [INSERT AverageLatency(us)=16.74] [READ AverageLatency(us)=16116.04] 
2014-07-04 18:22:08:523 80 sec: 1273831 operations; 12238.61 current ops/sec; [INSERT AverageLatency(us)=26.3] [READ AverageLatency(us)=15970.7] 
2014-07-04 18:22:18:524 90 sec: 1439678 operations; 16583.04 current ops/sec; [INSERT AverageLatency(us)=26.9] [READ AverageLatency(us)=11778.79] 
2014-07-04 18:22:28:525 100 sec: 1579131 operations; 13943.91 current ops/sec; [INSERT AverageLatency(us)=20.14] [READ AverageLatency(us)=13954.92] 
2014-07-04 18:22:38:526 110 sec: 1725409 operations; 14626.34 current ops/sec; [INSERT AverageLatency(us)=22.36] [READ AverageLatency(us)=13470.5] 
2014-07-04 18:22:48:527 120 sec: 1869064 operations; 14364.06 current ops/sec; [INSERT AverageLatency(us)=21.37] [READ AverageLatency(us)=13819.54] 
2014-07-04 18:22:58:527 130 sec: 1968294 operations; 9923 current ops/sec; [INSERT AverageLatency(us)=16.93] [READ AverageLatency(us)=19691.76] 
2014-07-04 18:23:08:528 140 sec: 2089302 operations; 12099.59 current ops/sec; [INSERT AverageLatency(us)=27.2] [READ AverageLatency(us)=16186.52] 
2014-07-04 18:23:18:529 150 sec: 2251244 operations; 16192.58 current ops/sec; [INSERT AverageLatency(us)=17.24] [READ AverageLatency(us)=12209.01] 
2014-07-04 18:23:28:530 160 sec: 2391015 operations; 13977.1 current ops/sec; [INSERT AverageLatency(us)=20.24] [READ AverageLatency(us)=13915.31] 
2014-07-04 18:23:38:530 170 sec: 2522314 operations; 13128.59 current ops/sec; [INSERT AverageLatency(us)=24.19] [READ AverageLatency(us)=14905.16] 
2014-07-04 18:23:48:531 180 sec: 2673173 operations; 15084.39 current ops/sec; [INSERT AverageLatency(us)=18.15] [READ AverageLatency(us)=13125.67] 
2014-07-04 18:23:58:532 190 sec: 2824282 operations; 15110.9 current ops/sec; [INSERT AverageLatency(us)=18.07] [READ AverageLatency(us)=13153.49] 
2014-07-04 18:24:08:532 200 sec: 2977769 operations; 15347.17 current ops/sec; [INSERT AverageLatency(us)=19.24] [READ AverageLatency(us)=12875.27] 
2014-07-04 18:24:18:533 210 sec: 3125756 operations; 14797.22 current ops/sec; [INSERT AverageLatency(us)=16.88] [READ AverageLatency(us)=13158.37] 
2014-07-04 18:24:28:534 220 sec: 3292920 operations; 16714.73 current ops/sec; [INSERT AverageLatency(us)=21.1] [READ AverageLatency(us)=11715.2] 
2014-07-04 18:24:38:535 230 sec: 3448610 operations; 15567.44 current ops/sec; [INSERT AverageLatency(us)=15.6] [READ AverageLatency(us)=12541.47] 
2014-07-04 18:24:48:536 240 sec: 3612202 operations; 16357.56 current ops/sec; [INSERT AverageLatency(us)=17.24] [READ AverageLatency(us)=12167.12] 
2014-07-04 18:24:58:536 250 sec: 3762239 operations; 15003.7 current ops/sec; [INSERT AverageLatency(us)=16.52] [READ AverageLatency(us)=13013.76] 
2014-07-04 18:25:08:537 260 sec: 3913115 operations; 15086.09 current ops/sec; [INSERT AverageLatency(us)=15.79] [READ AverageLatency(us)=13001.01] 
2014-07-04 18:25:18:538 270 sec: 4076872 operations; 16374.06 current ops/sec; [INSERT AverageLatency(us)=17.91] [READ AverageLatency(us)=11946.27] 
2014-07-04 18:25:28:539 280 sec: 4236828 operations; 15994 current ops/sec; [INSERT AverageLatency(us)=17.08] [READ AverageLatency(us)=12371.43] 
2014-07-04 18:25:38:540 290 sec: 4396398 operations; 15955.4 current ops/sec; [INSERT AverageLatency(us)=20.57] [READ AverageLatency(us)=12208.7] 
2014-07-04 18:25:48:541 300 sec: 4541738 operations; 14532.55 current ops/sec; [INSERT AverageLatency(us)=16] [READ AverageLatency(us)=13338.72] 
2014-07-04 18:25:58:605 310 sec: 4718410 operations; 17554.85 current ops/sec; [INSERT AverageLatency(us)=16.37] [READ AverageLatency(us)=11412.54] 
2014-07-04 18:26:08:606 320 sec: 4880808 operations; 16239.8 current ops/sec; [INSERT AverageLatency(us)=21.92] [READ AverageLatency(us)=11950.06] 
2014-07-04 18:26:18:606 330 sec: 4999361 operations; 11854.11 current ops/sec; [UPDATE AverageLatency(us)=396077.24] [INSERT AverageLatency(us)=12.48] [READ AverageLatency(us)=9880.03] [CLEANUP AverageLatency(us)=396259.29] 
2014-07-04 18:26:19:736 331 sec: 5000000 operations; 565.49 current ops/sec; [UPDATE AverageLatency(us)=371408] [INSERT AverageLatency(us)=15.09] [READ AverageLatency(us)=13162.44] [CLEANUP AverageLatency(us)=371427] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404523580
1404523580
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ let counter=1+1
+ read line
+ set +x
